L1Trigger/L1TTrackMatch/python/DisplacedVertexProducer_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+DisplacedVertexProducer = cms.EDProducer('DisplacedVertexProducer',
+  l1TracksInputTag = cms.InputTag("l1tTTTracksFromExtendedTrackletEmulation", "Level1TTTracks"),
+  l1TrackVertexCollectionName = cms.string("dispVertices"),
+  mcTruthTrackInputTag = cms.InputTag("TTTrackAssociatorFromPixelDigisExtended", "Level1TTTracks"),
+  ONNXmodel = cms.string("/afs/cern.ch/user/r/rmccarth/private/dispVert/CMSSW_14_0_0_pre3/src/L1Trigger/L1TTrackMatch/test/trackAndVert_NewTQMVA_model.onnx"),
```

Comment:
Similar to Claire's comment below this also needs to be in a cms-data repository, we have one for L1TTrackMatch here: https://github.com/cms-data/L1Trigger-L1TTrackMatch

Reply:
Just made a PR: https://github.com/cms-data/L1Trigger-L1TTrackMatch/pull/2